ui/src/templates/components/TemplateCard.tsx

```diff
@@ -0,0 +1,67 @@
+// Libraries
+import React, {PureComponent, MouseEvent} from 'react'
+import {withRouter, WithRouterProps} from 'react-router'
+
+// Components
+import {ResourceList} from 'src/clockface'
+
+// Actions
+
+// Types
+import {TemplateSummary} from '@influxdata/influx'
+
+// Constants
+import {DEFAULT_TEMPLATE_NAME} from 'src/templates/constants'
+
+interface Props {
+  template: TemplateSummary
+  onFilterChange: (searchTerm: string) => void
+}
+
+export class TemplateCard extends PureComponent<Props & WithRouterProps> {
+  public render() {
+    const {template} = this.props
+
+    return (
+      <ResourceList.Card
+        testID="template-card"
+        labels={() => null}
```

Comment:
this seems to be the default prop for labels in ResourceCard, is it necessary to define it here?

Reply:
removed.